src/main/java/com/icthh/xm/uaa/config/UaaAccessTokenConverterConfiguration.java

```diff
@@ -21,12 +21,15 @@
 import java.security.cert.CertificateException;
 import java.security.cert.CertificateFactory;
 import java.security.cert.X509Certificate;
+import java.security.KeyFactory;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.PKCS8EncodedKeySpec;
 
 import lombok.SneakyThrows;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.commons.lang3.StringUtils;
+import org.bouncycastle.util.encoders.Base64;
```

Comment:
@VovaViter 
do we really need Base64 from `bouncycastle` package? why do not use `java.util.Base64`?